fakes/fake_git_hub.go

```diff
@@ -111,14 +111,16 @@ type FakeGitHub struct {
 	}
 }
 
-func (fake *FakeGitHub) ListReleases() ([]*github.RepositoryRelease, error) {
+func (fake *FakeGitHub) ListReleases(tagNameRegex string) ([]*github.RepositoryRelease, error) {
 	fake.listReleasesMutex.Lock()
 	fake.listReleasesArgsForCall = append(fake.listReleasesArgsForCall, struct{}{})
 	fake.listReleasesMutex.Unlock()
 	if fake.ListReleasesStub != nil {
 		return fake.ListReleasesStub()
 	} else {
-		return fake.listReleasesReturns.result1, fake.listReleasesReturns.result2
+		matchedReleases, _ := resource.FilterReleasesByTagName(fake.listReleasesReturns.result1, tagNameRegex)
```

Comment:
Thanks for the feedback, I hadn't realised that the fakes were generated, however when using generated ones the regex filtering doesn't get applied. How would you rather we test filtering?